internal/acctest/authenticationpoliciessettings/authentication_policies_settings_test.go

```diff
@@ -0,0 +1,112 @@
+package acctest_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/acctest"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/provider"
+)
+
+// Attributes to test with. Add optional properties to test here if desired.
+type authenticationPoliciesSettingsResourceModel struct {
+	enableIdpAuthnSelection bool
+	enableSpAuthnSelection  bool
+}
+
+func TestAccAuthenticationPoliciesSettings(t *testing.T) {
+	resourceName := "myAuthenticationPoliciesSettings"
+	// No required resources, test empty resource
+	emptyResourceModel := authenticationPoliciesSettingsResourceModel{}
```

Comment:
This won't result in actual empty HCL being sent, because you are still setting the attributes in the HCL you provide in the testAccAuthenticationPoliciesSettings method. You need to provide HCL with no attributes set. The log settings test does this with some optional HCL it inserts based on a boolean.